test/Staking.ts

```diff
@@ -4,14 +4,15 @@ import { BigNumber } from "ethers";
 import { expect } from "chai";
 import { Staking } from "../types/Staking";
 
+
 describe("Staking contract", function () {
   let accounts: SignerWithAddress[];
   let contract: Staking;
   beforeEach(async () => {
     accounts = await ethers.getSigners();
 
     const contractFactory = await ethers.getContractFactory("Staking");
-    contract = (await contractFactory.deploy()) as Staking;
+    contract = (await contractFactory.deploy(4, 6)) as Staking;
```

Comment:
Please extract these magic numbers so they're explicit in the test

Reply:
Fixed 115588a7e5c6265e64aa2ce6755bf45a100913b9